src/main/java/co/inventorsoft/academy/emails/EmailConsole.java

```diff
@@ -0,0 +1,103 @@
+package co.inventorsoft.academy.emails;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailConsole {
+
+	static private final String EMAIL_REGEXP = "^[A-Z0-9._%+-]+@[A-Z0-9.-]+\\.[A-Z]{2,6}$";
```

Comment:
Try to keep declaration order regarding to Java Code Conventions.

Correct example:
**private** **static** final String EMAIL_REGEXP = "^[A-Z0-9._%+-]+@[A-Z0-9.-]+\\.[A-Z]{2,6}$";